Mathlib/MeasureTheory/Integral/IntervalIntegral.lean

```diff
@@ -971,6 +971,12 @@ theorem integral_Iic_sub_Iic (ha : IntegrableOn f (Iic a) μ) (hb : IntegrableOn
   exacts [measurableSet_Ioc, ha, hb.mono_set fun _ => And.right]
 #align interval_integral.integral_Iic_sub_Iic intervalIntegral.integral_Iic_sub_Iic
 
+theorem integral_Iic_add_Ioi (h_left : IntegrableOn f (Iic b) μ)
```

Comment:
Please also add the `Iio_add_Ici` version.

Reply:
done. This needed `Iio_disjoint_Ici`, which was not yet there. Let me know if it is preferable to do that in a separate pr.

Reply:
Which is usually preferred between `(h_left : IntegrableOn f (Iio b) μ) (h_right : IntegrableOn f (Ici b) μ)` and `Integrable f μ`?